components/MapOfCircles/MapOfCircles.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState, useRef } from "react";
```

Comment:
We no longer need to import `React` into our files. 

```suggestion
import  { useEffect, useState, useRef } from "react";
```